vim_template/langs/go/go.bundle

```diff
@@ -1,2 +1,3 @@
 "" Go Lang Bundle
 Plug 'fatih/vim-go', {'do': ':GoInstallBinaries'}
+Plug 'Valloric/YouCompleteMe'
```

Comment:
yes fix https://github.com/avelino/vim-bootstrap/pull/222/commits/d43d790857b33593e27ceb1e82637a0090935914 it will run install.py for C-family langs